profile/README.md

```diff
@@ -271,8 +271,47 @@ Please read the [Tracing section](https://docs.axoniq.io/reference-guide/extensi
 
 ## Bill of Materials
 
+The [Bill of Materials](https://github.com/AxonFramework/axon-bom) repository of Axon Framework is a dedicated dependency providing compatible versions of the [framework](#axon-framework) and [extensions](#extensions).
+Hence, by adding `axon-bom` to your dependency management system, you ensure the compatibility of Axon's dependencies.
+As such, we recommend you use this dependency whenever you combine Axon Framework with any of the extensions.
+
+The BOM is intended to release with every [Axon Framework](https://github.com/AxonFramework/AxonFramework) release.
+Hence, a 4.7.0 release of Axon Framework leads to a 4.7.0 release of the BOM.
+
 ## Inspector Axon
 
+The Axon Framework team designed [Inspector Axon](https://inspector.axoniq.io/) to further enable [Axon Framework](#axon-framework) applications with valuable data and tooling.
+
+Inspector Axon provides you with numerous benefits:
+- [Event Processor](https://docs.axoniq.io/reference-guide/axon-framework/events/event-processors) management tooling, like start, stop, [split, and merge](https://docs.axoniq.io/reference-guide/axon-framework/events/event-processors/streaming#splitting-and-merging-segments)
+- Automated monitoring of *all* your message handlers
+- Insights into the flow of your messages with valuable diagrams
+- The information required to deep-dive into the performance of your Axon Framework application
+
+By doing all this, we aim to teach the inner workings of Axon Framework to any user, enabling you the get the most out of the framework.
+
+The [how it works](https://inspector.axoniq.io/how-it-works) page helps you understand how Inspector Axon provides all this information.
+We refer you to [this](https://inspector.axoniq.io/legal) page if you (and your team) are curious to understand how we secure all this behavior.
+
 ## IntelliJ IDEA Plugin
 
-## AxonIQ Initializr
\ No newline at end of file
+The [Axon Framework IntelliJ Plugin](https://github.com/AxonFramework/IdeaPlugin) is a plugin you can add when using [IntelliJ IDEA](https://www.jetbrains.com/idea/).
```

Comment:
Should we link to the marketplace directly? https://plugins.jetbrains.com/plugin/18628-axon-framework

Reply:
Super smart! Will do